src/app/components/filter/filter-panel.jsx

```diff
@@ -48,14 +61,21 @@ const useStyles = makeStyles({
     display: 'grid',
     gridTemplateColumns: '1fr',
     alignItems: 'center',
-    borderLeft: '1px solid #599bde',
     paddingLeft: '16px',
   },
   filterSubHeaderContainer: {
     display: 'grid',
-    gridTemplateColumns: '1fr',
-    justifyItems: 'start',
-    gap: '8px',
+  },
+  clearResetContainer: {
```

Comment:
I think this is used to make the clear/reset stuff sticky, right? I think it works, but now the scroll has the actual primary buttons scroll away. I think that's even worse. Can we make those buttons sticky?

I didn't think this was going to be that hard; I thought we'd be able to just put the list of checkboxes inside a container that has a max height and let all the rest of the stuff in the filter panel not care about the checklist container

Reply:
Do you mean all the filters or just the sub heads?